packages/shared/src/utilities/color/isDark.ts

```diff
@@ -10,7 +10,14 @@ import { toShortRgba } from './toShortRgba';
  * @param {Object} Object of RGBA values
  * @returns {Boolean} Return if the color is dark
  */
-export const isDark = (color: Color): boolean => {
-    const parsedColor = tinycolor(toShortRgba(color));
+
+const isRgbaLongFormat = (value: Color) => {
+    const requiredKeys = ['red', 'green', 'blue'];
+    return typeof value === 'object' && requiredKeys.every((i) => value.hasOwnProperty(i));
+};
+
+export const isDark = (color: unknown): boolean => {
+    const inputColor = isRgbaLongFormat(color as Color) ? toShortRgba(color) : (color as ColorInput);
+    const parsedColor = tinycolor(inputColor);
     return parsedColor.isDark() || (parsedColor.getAlpha() > 0.25 && parsedColor.getAlpha() < 1);
```

Comment:
It seems that tinycolor cannot handle our color format correctly. isDark always returns "true" even if you select a white background for warnings.

![image](https://user-images.githubusercontent.com/22001580/199235667-6a616f79-d94c-4ce8-b5f0-611e4d940dcb.png)

No matter what you select as background color, the font color is always white...

Reply:
Good catch! Fixed.